src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.unit.spec.jsx

```diff
@@ -308,6 +308,24 @@ describe('VAOS <ConfirmedAppointmentDetailsPage> with VAOS service', () => {
       start: futureDate.format(),
       status: 'cancelled',
       cancelationReason: { coding: [{ code: 'pat' }] },
+      location: {
+        id: '983GC',
+        type: 'appointments',
+        attributes: {
+          id: '983GC',
+          vistaSite: '983GC',
+          name: 'Cheyenne VA Medical Center',
+          lat: 39.744507,
+          long: -104.830956,
+          phone: { main: '307-778-7550' },
+          physicalAddress: {
+            line: ['2360 East Pershing Boulevard'],
+            city: 'Cheyenne',
+            state: 'WY',
+            postalCode: '82001-5356',
+          },
+        },
+      },
```

Comment:
Could not successfully create a mock facility in version 2. Alternatively, added the facility details in the appointment attribute.